### ISSUE TITLE
Locate error for unbound value

### ISSUE THREAD

**Issue by vch9** [jobjo/popper#52]
Hi!

The following code:
```ocaml
type t = int [@@deriving popper]
```
Produces that error:
```ocaml
File "_none_", line 1: 
Error: Unbound value pp
```
I haven't look deeply on how you integrate the `pp`, but maybe it could be located
on the type declaration structure ?

**Comment by vch9** [comment]
I think it would also be nice to derive `show` and `ord` when you wan't to derive `popper`.
I don't know if it's technically doable, it could also hide previous definition of a pretty-printter if the
user doesn't know that `popper` derives `show`..

**Comment by jobjo** [comment]
Yeah, I also don't know if it's possible to make it to make `[@@deriving popper]` and expand it to `[@@deriving show, ord, popper]`. Will check.

Alternatively we could add our own `show`, `ord` derivers and remove the dependency on `ppx_deriving_show/ord`.

**Comment by vch9** [comment]
> Alternatively we could add our own `show`, `ord` derivers and remove the dependency on `ppx_deriving_show/ord`.

You mean require the user to write `show` and `ord` or to derivate yourself both of them ?

**Comment by jobjo** [comment]
> You mean require the user to write show and ord or to derivate yourself both of them ?

I mean implement ppxs for both so that when you derive `popper` you also get the `pp` and `compare` functions. That also implies removing the dependency on `ppx_deriving`.
